frontend/src/app/store/selectors/auth-page.selectors.ts

```diff
@@ -0,0 +1,27 @@
+import { createFeatureSelector, createSelector } from '@ngrx/store';
+
+import { AuthState } from 'src/app/types/states/authState';
+import { Token } from 'src/app/types/auth/token';
+import { User } from 'src/app/types/user/user';
+
+export const selectAuthState = createFeatureSelector<AuthState>('authState');
+
+export const selectUser = createSelector(
+    selectAuthState,
+    (state) => state.user
+);
+
+export const selectToken = createSelector(
+    selectAuthState,
+    (state) => state.token
+);
+
+export const selectUserId = createSelector(
+    selectUser,
+    (state: User) => state.id
```

Comment:
I do not see 'id' in state declaration. State contains user nad token only

you are accessing not a store, rename variable from state to suer

Reply:
Fixed